starboard/shared/win32/posix_emu/dirent.cc

```diff
@@ -180,18 +184,25 @@ DIR* opendir(const char* path) {
 }
 
 int closedir(DIR* dir) {
+  if (!dir)
+    errno = EBADF;
+  return -1;

```

Comment:
This looks like you maybe missed the braces around `if(!dir)` here. I don't think there should be an unconditional `return -1`

Reply:
Gah, good catch! Ty